src/components/cursor-controller.js

```diff
@@ -93,8 +93,8 @@ AFRAME.registerComponent("cursor-controller", {
     this.data.playerRig.addEventListener(this.data.primaryUp, this._handlePrimaryUp);
     this.data.playerRig.addEventListener(this.data.grabEvent, this._handlePrimaryDown);
     this.data.playerRig.addEventListener(this.data.releaseEvent, this._handlePrimaryUp);
-    this.data.playerRig.addEventListener("gamepadbuttondown", this._handlePrimaryDown);
```

Comment:
I'm guessing these were unused?

Reply:
this was from my previous attempt when trying to use `gamepad-controls`. I left them in because I knew i was going to reuse them for this PR, but wound up deciding on a different event name